todo_app/data/trello_items.py

```diff
@@ -0,0 +1,51 @@
+import requests
+import os
+
+from todo_app.item import Item
+
+auth = {
+      'key': os.getenv('TRELLO_KEY'), 
+      'token': os.getenv('TRELLO_TOKEN'),
+    }
+
+def get_items():
+    response = requests.get('https://trello.com/1/boards/{board}/cards'.format(board=os.getenv('TRELLO_BOARD')), params=auth)
+    cards = response.json()
+    items = []
+    for card in cards:
+        item = Item.from_trello_card(card, map_list_id_to_name(card['idList']))
+        items.append(item)
+    return items
+
+def map_list_id_to_name(listID):
+    lists = get_lists()
+    for list in lists:
+        if listID == list['id']:
+            return list['name']
+    return ''
+
+def add_item(title):
+    boardLists = get_lists()
+    todoBoardIndex = find_list_by_name('To Do', boardLists)

```

Comment:
Similarly on could we speed this up, the two simplest approaches are likely to be:
* Give up some of your flexibility, just get the developer to pass the relevant id in as an environment variable in the first place!
* Calculate this, but cache it somewhere, so e.g. something like below (untested)
```
my_lists = []
def get_list_id(list_name):
    if len(my_lists) == 0:
            my_lists += get_lists()
    for a_list in my_lists:
         if a_list['name'] == list_name:
             return a_list['id']
    # deal with mishits
```